src/dataIntegration/transcriptomics/extractGPRs.m

```diff
@@ -0,0 +1,120 @@
+function [parsedGPR,corrRxn] = extractGPRs(model)
+%Maps the GPR rules of the model to a specified format that is used by the 
+%model extraction methods 
+%
+%INPUTS
+%
+%   model           input model (COBRA model structure)
+%
+%OUTPUTS
+%
+%   parsedGPR       cell array describing the possible combination of gene needed for each
+%                   reactions in the model without using "AND" and "OR" logical rule
+%
+%   corrRxn         cell array containg the reaction names associated to parsedGPR
+%
+%
+% originally written in createTissueSpecificModel.m
+% annotated and modified by A. Richelle May 2017
+
+    warning off all
+
+    parsedGPR = [];
+    corrRxn = [];
+    cnt = 1;
+
+    for i = 1:length(model.rxns)           
+        if length(model.rules{i}) > 1
+            % Parsing each reactions GPR containing "OR" rule
+
+            [parsing{1,1},parsing{2,1}] = strtok(model.rules{i},'|');           
+            for j = 2:1000
```

Comment:
while this j is probably sufficient, shouldn't it depend on the actual number of tokens?